selftests/lint.sh

```diff
@@ -9,22 +9,22 @@ PYLINT=$(which pylint-3 2>/dev/null || which pylint)
 
 # Those are our main packages, we should follow the .pylintrc file with all
 # enabled by default. Some are disabled, we are working to reduce this list.
```

Comment:
We should update this comment too. replacing  the "packages" word.

Reply:
I missed that, thanks.